### DISCUSSION TITLE
Silence OSQP

### DISCUSSION THREAD

**Discussion by RudolfReiter** [casadi/casadi#3174] [General]
Dear all, dear Joris,
how can one silence (verbose=False for all terminal prints) OSQP when interfaced with casadi conic in python?
Best,
Rudolf

**Comment by RudolfReiter** [comment]
Found the solution by trial and error:
opti.solver('osqp', { "osqp":{"verbose":False}})

**Reply by maximiliandegner** [reply]
I faced the same question but when using qpsol('solver', 'osqp', qp_struct, qp_opt). 
Using qp_opt={ "osqp":{"verbose": False}} as @RudolfReiter posted, silenced OSQP in qpsol too.
